website/homepage/Dockerfile

```diff
@@ -14,15 +14,15 @@ RUN npm i pnpm@6 -g
 WORKDIR /gradio/ui
 ENV NODE_OPTIONS="--max-old-space-size=4096"
 RUN pnpm i
-RUN pnpm build:website
+RUN pnpm build
 WORKDIR /gradio
 COPY ./gradio ./gradio
 COPY ./requirements.txt ./requirements.txt
 COPY ./setup.py ./setup.py
 COPY ./MANIFEST.in ./MANIFEST.in
 COPY ./README.md ./README.md
 COPY ./test ./test
-RUN python setup.py install
```

Comment:
For some reason, this doesn't work on an M1. You end up pulling a pre-release of `orjson`

```
#0 6.147 Processing orjson-3.7.5.tar.gz
#0 6.200 error: Couldn't find a setup script in /tmp/easy_install-xdiqoz9x/orjson-3.7.5.tar.gz
```